nimble/host/audio/syscfg.yml

```diff
@@ -0,0 +1,21 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#  http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+syscfg.defs:
+
+syscfg.logs:
```

Comment:
I think we don't need syscfg.logs here. And under defs, I'd move BLE Audio configs from `host/syscfg.yml`: `BLE_MAX_BIG` and `BLE_MAX_BIS`. And maybe ISO related configs too? I mean `BLE_ISO`, `BLE_ISO_BROADCAST_SOURCE` and so on. What do you think @andrzej-kaczmarek ? This way we'll limit bloating of host syscfg

Reply:
Once the code will be moved to `host/syscfg.yml` in subsequent PR, I'll place the configs as you suggested.